server/helpers.py

```diff
@@ -9,16 +9,22 @@ def register_parser():
     registerparser.add_argument('username', help = 'This Field Cannot be blank', required=True)
     registerparser.add_argument('email', help = 'This Field Cannot be blank', required=True)
     registerparser.add_argument('password', help = 'This Field Cannot be blank', required=True)
+    registerparser.add_argument('firstName', help = 'This Field Cannot be blank', required=True)
+    registerparser.add_argument('lastName')
+    registerparser.add_argument('gender')
+    registerparser.add_argument('phone')
+    registerparser.add_argument('branch')
+    registerparser.add_argument('year')
     return registerparser
 
 def login_parser():
     loginparser = reqparse.RequestParser()
-    loginparser.add_argument('email', help='This Field Cannot be blank', required=True)
+    loginparser.add_argument('username', help='This Field Cannot be blank', required=True)
     loginparser.add_argument('password', help='This Field Cannot be blank', required=True)
     return loginparser
 
 # helper for userRegistration
-def userRegister(username, email, password):
+def userRegister(username, email, password, firstName, lastName, gender, phone, branch, year):
     check_username = users.User.objects(username=username).first()
```

Comment:
Check username function is created but never used
it should  be checked wthether a user with the same username exists or not beacuse we are using username to login the user

If user with the same username exists Raise an exception username already exists.

Reply:
ok

Reply:
check_username function is in use. it is not displayed here because GitHub is not comparing it with change

Reply:
ok